### PR TITLE
Create tests for attribute view classes

### PR DESCRIPTION
**Pull request by flagbird**
This part of the bundle is the most complex one. This PR targets some of the basic functionalities and increase the coverage, but isn't covering by far everything relevant. Further development on this feature will also improve the edge cases in time.

### CODE REVIEW

File: jest/public/reference-entity/attribute/table/modal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    attribute?: TableAttribute;
+    rights?: RightsType;
+} = {}) {
+
+    options = {
+        attribute: createDefaultAttribute(),
+        saveTable: jest.fn(),
+        rights: accessRightsTrue,
+        ...options
+    }
```
Comment (mralexandernickel) [inline jest/public/reference-entity/attribute/table/modal.test.tsx:193]: i like!

File: src/Resources/public/reference-entity/attribute/table/view.tsx [hunk 2]
```diff
@@ -6,6 +6,17 @@ import TableAttributeModal from './modal';
 import { connect } from 'react-redux';
 import $ from 'jquery';
 
+export type RightsType = {
```
Comment (mralexandernickel) [inline src/Resources/public/reference-entity/attribute/table/view.tsx:9]: Why not simply `Rights`? I mean its obvious that this is a type...

Comment (flagbird) [reply src/Resources/public/reference-entity/attribute/table/view.tsx:9]: No specific reason. I wanted to prevent confusion about `rights` and `Rights`.

**Review by mralexandernickel — APPROVED** [review]

**Comment by flagbird** [comment]
Thank you for the review @mralexandernickel
